pkgs/development/tools/google-java-format/default.nix

```diff
@@ -17,11 +17,17 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out/{bin,share/google-java-format}
-    install -D ${src} $out/share/google-java-format/google-java-format.jar
-
-    makeWrapper ${jre}/bin/java $out/bin/google-java-format \
-      --add-flags "-jar $out/share/google-java-format/google-java-format.jar"
+    mkdir -p $out/{bin,share}
+    install -D ${src} $out/share/google-java-format-${version}-all-deps.jar
```

Comment:
You removed the directory `google-java-format`. The documentation suggests:

> Private JARs should be installed in a location like `$out/share/package-name`.

Reply:
Restored it.

Now that I know there are guidelines for this, maybe I'll clean up some of the other java packages, since I've come across so many variations in the repo.